dask/dataframe/io/tests/test_csv.py

```diff
@@ -1081,6 +1081,10 @@ def test_index_col():
         except ValueError as e:
             assert "set_index" in str(e)
 
+        df = pd.read_csv(fn, index_col=False)
+        ddf = dd.read_csv(fn, blocksize=30, index_col=False)
+        assert_eq(df, ddf, check_index=False)
```

Comment:
Why are we setting check_index to False? Otherwise good

Reply:
Test implementation detail. With `check_index=True` the test fails as:

```python
E   AssertionError: DataFrame.index are different
E   
E   DataFrame.index values are different (92.30769 %)
E   [left]:  Int64Index([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12], dtype='int64')
E   [right]: Int64Index([0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 3], dtype='int64')
```

The idx are mismatched after dask collates each worker's df into the final ddf. This happens whether or not we pass `index_col=False` to both `read_csv` functions. It also doesn't really matter for the purposes of this test -- by passing `index_col=False` we're telling pandas to explicitly ignore any idx in the file and provide its own idx. The fact that their outputs differ is a dask implementation detail. I guess we could throw a malformed CSV test in here and confirm it both loads properly and the dask/pd outputs are the same... but that seems more like a test that should live in pandas.

Also, you can see the same pattern `check_index=False` in a couple other tests in this file, e.g. `test_header_int`, `test_read_csv_names_not_none`, `test_to_csv_with_get`.